src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -3,26 +3,69 @@
 import nextstep.subway.line.application.LineService;
 import nextstep.subway.line.dto.LineRequest;
 import nextstep.subway.line.dto.LineResponse;
+
+import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.net.URI;
+import java.util.List;
+
+import javax.persistence.EntityNotFoundException;
 
 @RestController
 @RequestMapping("/lines")
 public class LineController {
-    private final LineService lineService;
+	private final LineService lineService;
+
+	public LineController(final LineService lineService) {
+		this.lineService = lineService;
+	}
+
+	@PostMapping
+	public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+		LineResponse line = lineService.saveLine(lineRequest);
+		return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
+	}
+
+	@GetMapping
+	public ResponseEntity<List<LineResponse>> showLiens() {
+		return ResponseEntity.ok().body(lineService.findAllLines());
+	}
+
+	@GetMapping("/{lineID}")
+	public ResponseEntity<LineResponse> showLine(@PathVariable("lineID") long lineID) {
+		return ResponseEntity.ok().body(lineService.getLine(lineID));
+	}
+
+	@PutMapping("/{lineID}")
+	public ResponseEntity updateLine(@PathVariable("lineID") long lineID,
+		@RequestBody LineRequest lineRequest) {
+		this.lineService.updateLine(lineID,lineRequest);
+		return ResponseEntity.ok().build();
+	}
+
+	@DeleteMapping("/{lineID}")
+	public ResponseEntity deleteLine(@PathVariable("lineID") long lineID) {
```

Comment:
중요한 부분은 아니지만

```suggestion
	public ResponseEntity deleteLine(@PathVariable("lineId") long lineId) {
```

일반적으로 카멜 케이스 사용 시 lineID가 아닌 lineId로 많이 사용했던 것 같습니다.

Reply:
네 알겠습니다. 수정하겠습니다 :)